docs/RouteRequest.md

```diff
@@ -192,22 +192,31 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The minimum extra time after exiting a public transport vehicle.
+The extra time after exiting a public transport vehicle.
 
 The slack is added to the time when going from the transit vehicle to the stop.
+This also influences the total time it takes to transfer. See `transferSlack` for more details about
+how the total transfer time is calculated.
+
 
 <h3 id="rd_boardSlack">boardSlack</h3>
 
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The boardSlack is the minimum extra time to board a public transport vehicle.
+The extra time before boarding a public transport vehicle.
+
+The slack is added to the time when going from the stop (offboard) to onboard a transit
+vehicle. It is useful for suggesting passengers arrive at the stop a little earlier than they strictly
+need to to allow for unexpected circumstances.
 
-The board time is added to the time when going from the stop (offboard) to onboard a transit
-vehicle.
+This is similar to `transferSlack`, except that this also applies to the first
+transit leg in the trip.
 
-This is the same as the `transferSlack`, except that this also applies to to the first
-transit leg in the trip. This is the default value used, if not overridden by the `boardSlackList`.
+This also influences the total time it takes to transfer. See `transferSlack` for more details about
+how the total transfer time is calculated.
+
+This is the default value used, if not overridden by the `boardSlackList`.
```

Comment:
I don't know if we should refer to `boardSlackList` here, maybe a more generic mention about it being overriden with a query parameter is better. Also, `boardSlackForMode` could be mentioned here.

Reply:
I removed boardSlackList because that appears to be only used in the Transmodel API.